DataFormats/SiStripCluster/interface/SiStripApproximateClusterv1.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DATAFORMATS_SISTRIPAPPROXIMATECLUSTERv1_H
+#define DATAFORMATS_SISTRIPAPPROXIMATECLUSTERv1_H
+
+#include <numeric>
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+
+class SiStripApproximateClusterv1  {
+public:
+  
+  SiStripApproximateClusterv1() {}
+  explicit SiStripApproximateClusterv1(const SiStripCluster& cluster){
+    barycenter_=static_cast<uint16_t>(cluster.barycenter());
+    width_=cluster.size();
+    avgCharge_ = static_cast<uint8_t>(cluster.charge()/cluster.size());
+  }
+  
+  explicit SiStripApproximateClusterv1(uint8_t avgCharge, uint16_t barycenter, uint8_t width):avgCharge_(avgCharge) {
+    barycenter_ = barycenter;
+    width_ = width;
+    if(width_>0x3F) width_=0x3F;
+  }
+
+  uint16_t barycenter() const {return barycenter_;}
```

Comment:
I think one can store the baricenter in tenth of strips

Reply:
one can have two methods
````
uint16_t rawBarycenter()
float barycenter() { return 0.1f&float(awBarycenter());}

Reply:
I like the idea. Implementing. Also the barycenter should be scaled by 10 (being provocative, we can have up to 1/85 resolution). On the other hand, not sure what will be the compression factor.

Reply:
if you multiply by 8 you will get 3 more bits. by 16 4bits. I would not count to much on root compression of zeros...

Reply:
Did some comparison and I see that indeed increasing the resolution, will give the same uncompressed size (as expected) but the compressed size (ZLIB 7) is different. We are talking of just few % but it should be considered. I propose then to leave this version as is and create another version with an higher resolution. 

X | Uncompressed (Bytes) | Compressed (Bytes) | Size Increase %

1 | 432551 | 266686 | 0
2 | 432551 | 278681 | 4.49779891
4 | 432551 | 288405 | 8.14403456
8 | 432551 | 296358 | 11.1261934
16 | 432551 | 302937 | 13.5931395